react-delegationdashboard/src/components/PendingUndelegatedListView/UndelegatedValueRow.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { useEffect } from 'react';
+import { UndelegatedValueType } from '../../helpers/types';
+import moment from 'moment';
+import { Delegation } from '../../contracts';
+import { useContext } from '../../context';
+
+const UndelegatedValueRow = ({ undelegatedValue: value, index }: { undelegatedValue: UndelegatedValueType; index: number }) => {
+
+    const { dapp, delegationContract } = useContext();
+    const [isDisabled, setIsDisabled] = React.useState('disabled');
```

Comment:
Either make `isDisabled` boolean, and have a conditional class, or you can leave it as a string, but call the hooks `disabled` and `setDisabled`